pkg/destroy/gcp/policybinding.go

```diff
@@ -2,20 +2,26 @@ package gcp
 
 import (
 	"context"
+	"fmt"
 	"strings"
 
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
-	resourcemanager "google.golang.org/api/cloudresourcemanager/v1"
+	resourcemanager "google.golang.org/api/cloudresourcemanager/v3"
 	"k8s.io/apimachinery/pkg/util/sets"
 )
 
+const (
+	// projectNameFmt is the format string for project resource name.
+	projectNameFmt = "projects/%s"
```

Comment:
This is now defined multiple times, consider moving somewhere that both parts of the package can use it.

Reply:
suggestion incorporated.